data/strings/strings.txt

```diff
@@ -15779,7 +15789,7 @@
     tr = Bir açıklama yazın (metin veya html)
     uk = Додайте опис (текст чи html)
     vi = Đánh vào một mô tả (văn bản hoặc html)
-    zh-Hans = 请添加说明(文字或html)
+    zh-Hans = 请添加说明（文字或html）
```

Comment:
Trailing space here.

Reply:
So just need to change the text by adding @@ -15779,7 +15789,7 @@ in first line?